### ISSUE TITLE
[Programmatic Access - Bot Framework Web Chat - Card Breakfast]: 'Breakfast' text is visible as heading but not defined programmatically.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#5341]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
Screen reader users will face difficulty in understanding the structure of the page while navigating using the h shortcut key and cannot easily understand the context of the dialog if the main heading of screen is not programmatically defined.

### Note: 
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated, Suggested action

### What is the public URL for the website?

 https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Card Breakfast word in type your message text box.
4. Navigate to the results.
5. Verify whether heading is defined or not.

### What do you expect?

'Breakfast' text is visible as heading and should be programmatically defined as heading.

### What actually happened?

'Breakfast' text is visible as heading but not defined programmatically.
Observation: There are no headings defined under the Bot framework web chat screen.

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/23a35de1-517f-4d37-a3ac-6a616d57dfb4)

### MAS reference

MAS 1.3.1 – Info and Relationships

### WCAG reference

1.3.1 Info and Relationships

### Note:
Same issue repro throughout the application.

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev2;#HCL;#Accessibility;#AILimited;#Benchmark;#BotFrameworkWebChat-Oct24;#MAS1.3.1;#Win11-Edge(Chromium);

**Comment by Kirank915** [comment]
@OEvgeny This issue is marked as external to Adaptive cards, could you please confirm if your team will fix the issue, or we need to share this issue to adaptive cards team?

**Comment by OEvgeny** [comment]
This can be improved by adjusting the card payload, so this is a bot authoring issue. We'll keep this in mind for further bots, but no further action is required.
